CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ if(NOT DEFINED COUCHBASE_CXX_CLIENT_MASTER_PROJECT)
   endif()
 endif()
 
+option(COUCHBASE_CXX_CLIENT_COLUMNAR "Build with Columnar additions" FALSE)
```

Comment:
We need CI build with this option to track regressions during future changes.

Reply:
Agreed -- added a path to build Columnar 'things' in the latest patchset.